packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -157,9 +157,10 @@ export interface NodegroupOptions {
    */
   readonly subnets?: SubnetSelection;
   /**
-   * The AMI type for your node group.
+   * The AMI type for your node group. If you explicitly specify the launchTemplate with custom AMI, do not specify this property, or
+   * the node group deployment will fail. In other cases, you will need to specify correct amiType for the nodegroup.
    *
-   * @default - auto-determined from the instanceTypes property.
+   * @default - undefined.
```

Comment:
This should be something like:
`auto-determined from the instanceTypes property when launchTemplateSpec property is not specified`